src/pages/attendance-marking-page.js

```diff
@@ -0,0 +1,110 @@
+import { Typography } from "@mui/material";
+import { Box } from "@mui/system";
+import { Button, TextField, Link } from "@mui/material";
+import * as React from "react";
```

Comment:
We don't need to import all from the react right? I think we can remove this import.

Reply:
removed the unnecessary imports